cics-bundle-maven-plugin/src/main/java/com/ibm/cics/cbmp/BundleDeployMojo.java

```diff
@@ -100,7 +100,7 @@
 	private String classifier;
 	
 	/**
-	 * The full URL of the endpoint.
+	 * The URL to the the CMCI connection the CICS bundle deployment API is available on. For example `https://yourcicshost.com:9080`.
```

Comment:
Got a double-'the'. Also, not sure about 'CMCI connection'... surely you make a connection to the URL?